packages/tools/tsconfig.json

```diff
@@ -5,8 +5,11 @@
         "esModuleInterop": true,
         "module": "CommonJS",
         "moduleResolution": "Node",
-        "rootDir": "./",
-        "outDir": "./dist"
+        "outDir": "./dist",
+        "paths": {
+            "@dd/tools/*": ["./src/*"],
+            "@dd/core/*": ["../core/src/*"]
+        }
```

Comment:
This config is slightly different from the other workspaces because we use `ts-node` to run the CLI.

I'd love to have the exact same config everywhere, but it bite me a few times already trying to unify this. 
Taking me hours every time I try to find a way, always reverting everything in the end.

Open to discuss if you have a very clear vision of it.